src/snowflake/cli/api/utils/definition_rendering.py

```diff
@@ -232,6 +239,26 @@ def _validate_env_section(env_section: dict):
             )
 
 
+def _get_referenced_vars_in_definition(
+    template_env: TemplatedEnvironment, definition: Definition
+):
+    referenced_vars = set()
+
+    def find_any_template_vars(element):
+        referenced_vars.update(template_env.get_referenced_vars(element))
+
+    traverse(definition, visit_action=find_any_template_vars)
+
+    return referenced_vars
+
+
+def _template_version_warning():
+    cc.warning(
+        "Ignoring template pattern in project definition file.\n"
+        "Update project definition version to 1.1 or later to enable template expansion."
```

Comment:
Why did you end up deciding not to mention snowflake.yml? I thought it would be useful for a user to be guided to the solution.

Reply:
what if they're using snowflake.local.yml?

Reply:
I think we can count on the user to understand that changes that apply to `snowflake.yml` can also apply to `snowflake.local.yml`. Having `definition_version` specified here may help too as they can simply search for that string in their project directory.

Reply:
@sfc-gh-cgorrie My thoughts exactly